polynote-frontend/style/styles.less

```diff
@@ -1190,6 +1190,15 @@ body {
         padding-right: 1em;
       }
 
+      .htmltext {
+        max-width: 100%;
+        box-sizing: border-box;
+
+        img, svg {
```

Comment:
Not so sure about this – doesn't this mean _all_ images in HTML output will be 100% width?

Reply:
This is actually no longer necessary after the change to use `kernel.display`, I'll remove. Thanks for the catch!